app/assets/javascripts/app/views/help_view.js

```diff
@@ -14,7 +14,7 @@ app.views.Help = app.views.StaticContentView.extend({
 
   initialize : function() {
     this.GETTING_HELP_SUBS = {
-      getting_started_a: {tutorial_series: this.linkHtml("http://diasporafoundation.org/getting_started/sign_up", Diaspora.I18n.t("getting_started_tutorial"))},
+      getting_started_a: {tutorial_series: this.linkHtml("https://diasporafoundation.org/getting_started/sign_up", Diaspora.I18n.t("getting_started_tutorial"))},
```

Comment:
- Identifier 'getting_started_a' is not in camel case.
- Identifier 'tutorial_series' is not in camel case.

Reply:
I think, this can be ignored, since the identifier was not changed

Reply:
If you touch it, you own it. Which, by the way, is exactly the reason why I'm insisting so hard to only change things where it's needed. As annoying as it is, if we fix the existing codestyle issues step by step, we'll eventually end up with a consistent codestyle everywhere. Here, the scope is minimal - renaming those variable names and renaming their usages in one single template file is enough.

Reply:
Whats about 47 language files these identifiers are also referenced? 
Is it a good idea just to change this identifier and let the direct 'neighbors' untouched? 

<img width="318" alt="Bildschirmfoto 2021-02-21 um 18 05 24" src="https://user-images.githubusercontent.com/501326/108632461-60d9a400-746f-11eb-82c9-37cb34f35d54.png">

Reply:
These keys have no reference to any language file. They're merely assignments to variables, they're passed as the variable object [here](https://github.com/diaspora/diaspora/blob/1f329991259dcaf201d3b0ca1e5ca3be306ce10b/app/assets/javascripts/app/views/help_view.js#L67), and as long as you change them both here and [in the template they're used](https://github.com/diaspora/diaspora/blob/1f329991259dcaf201d3b0ca1e5ca3be306ce10b/app/assets/templates/faq_getting_help_tpl.jst.hbs), you're fine. That's just an assignment for later use in the same unit, not relevant to what's stored in the language file.

`Diaspora.I18n.t()` is the function actually loading things from the language file. Changing those would be very bad indeed, but the bot is not complaining about that. :) 

> Is it a good idea just to change this identifier and let the direct 'neighbors' untouched?

It's not a good idea, no! Most likely the bot would yell at you anyway, because you're changing context lines.

Reply:
Ah wait, you're talking about `tutorial_series`, not `getting_started_a`. I actually completely missed that.

Yeah, you're right, that's bad. Okay, fine, let's **not** change the keys then, but please enclose the entire block with a 

```js
/* eslint-disable camelcase */
...
/* eslint-enable camelcase */
```

To declare that we're violating camelcase here on purpose, and to avoid further but complaints for changes in the future! :)